src/distribute_cases.py

```diff
@@ -0,0 +1,74 @@
+import configparser
+from df2gspread import gspread2df as g2d
+import numpy as np
+from os.path import expanduser, isfile
+import pandas as pd
+from random import sample
+
```

Comment:
Disclaimer: 💅

Python style guide propose [a convention to organize the `import` section](https://www.python.org/dev/peps/pep-0008/#imports). In a nutshell first we have native modules, then modules installed via `pip`, and then your application's modules. And on top of that, `import` comes before `from … import …`.

That said, I would have written:

```python
import configparser
from os.path import expanduser, isfile
from random import sample

import numpy as np
import pandas as pd
from df2gspread import gspread2df as g2d
```

But… well… again: 💅